### PR TITLE
Create ankushtripathii.json

### PR DESCRIPTION
**Pull request by ankushtripathii**
My personal profile

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1720"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Please move your file to the `public/data` directory.
Let us know if you need any help with that! 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: ankushtripathii.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Ankush Tripathi",
+    "type": "personal",
+    "bio": "I am Ankush Tripathi, currently pursuing my engineering degree in computer science from Atharva college of engineering. I am a tech enthusiast. I am always open to collaborating on projects and innovative/disruptive ideas",
```
Comment (shivam-sharma7) [inline ankushtripathii.json:4]:
```suggestion
    "bio": "I am Ankush Tripathi, currently pursuing my engineering degree in computer science from Atharva college of engineering. I am a tech enthusiast. I am always open to collaborate on projects and innovative/disruptive ideas",
```

Comment (shivam-sharma7) [reply ankushtripathii.json:4]: @ankushtripathii  change 'collaborating' to 'collaborate'

File: ankushtripathii.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "avatar": "https://github.com/ankushtripathii.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ankushtripathii",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/imalphaorionis?t=3IDp2LyC7wcgWujDy-eJbA&s=09",
```
Comment (kumarsonsoff3) [inline ankushtripathii.json:14]: This link will be more dynamic and simple! 
```suggestion
        "url": "https://twitter.com/imalphaorionis",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
- Please move your file to `public/data`. 
- Also, update your Twitter link!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant, Not maintainable

**Comment by ankushtripathii** [comment]
> * Please move your file to `public/data`.
> * Also, update your Twitter link!
> 
> Let us know if you need any help! __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Not maintainable

**Comment by ankushtripathii** [comment]
> * Please move your file to `public/data`.
> * Also, update your Twitter link!
> 
> Let us know if you need any help! __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix  |  **Tags:** Not maintainable

how to move file to `public/data `.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 
Simply go to your Editor, `Ctrl + X` from base folder & paste it to the folder you wanted to!! Just it is!!
I'm not sure if there is a way using the GitHub UI!

> **Warning**
>In this one, I've moved it to the required location, so no need to do it again!!

**Pull request merged**